### DISCUSSION TITLE
cn/2024/04/12/try/

### DISCUSSION THREAD

**Discussion by giscus** [hongtaoh/hongtaoh.github.io#159] [Announcements]
# cn/2024/04/12/try/

https://hongtaoh.com/cn/2024/04/12/try/

**Comment by residualsun1** [comment]
> 我想说的是，当我们有什么愿望，有什么梦想的时候，与其只停留在想的阶段，不如跨出去，去尝试。很多时候，甚至绝大部分时候，你的尝试会失败。但，如果尝试一下并没有什么损失，为什么不试一下。有一句话的大意是，当你知道你想去哪里时，全世界都会为你让路。是真的。

我有一位同学似乎也是这样，不论参加什么学术会议或活动，或者是遇到哪位领域的带头人，她永远是最敢发言、最敢提问的一个，反倒是身边自认为学历背景不好的其他同学，会有许多自卑、自己劝退自己的想法。我从大一看着那位同学到大三变得越来越自信，整个人气质也都不一样了。这样的人很有吸引力，还有一股勃勃的生命力，包括博主。

**Reply by hongtaoh** [reply]
你也可以成为你想成为的人，不用去羡慕别人
